frame/democracy/README.md

```diff
@@ -0,0 +1,135 @@
+# Democracy Pallet
+
+- [`democracy::Config`](https://docs.rs/pallet-democracy/latest/pallet_democracy/trait.Config.html)
+- [`Call`](https://docs.rs/pallet-democracy/latest/pallet_democracy/enum.Call.html)
+
+## Overview
```

Comment:
is it considered to change democrasy as lot? is it possible to reuse parts of that code base instead of copies? can we have reference to its doc instead of copy? locally only name difference?

Reply:
It's based on democracy but generalize to MultiCurrency instead of a single Currency I believe

Reply:
yep, that should be in readme.

Reply:
> @aettran what clickup numer this pr is related?

https://app.clickup.com/t/1rpbzkw

Reply:
in click up there is button with github icon and it, when added to  headers, allow to bin clickup to github.

Reply:
Task id:  `CU-1rpbzkw`. Will add it to my headers.